cocotb/config.py

```diff
@@ -97,6 +99,50 @@ def help_vars_text():
     return helpmsg
 
 
+def lib_name(interface, simulator):
+
+    interface_name = interface.lower()
+    supported_interfaces = ["vpi", "vhpi", "fli"]
+    if interface_name not in supported_interfaces:
+        raise ValueError("Wrong interface used. Supported: " + ", ".join(supported_interfaces))
+
+    simulator_name = simulator.lower()
+    supported_sims = ["icarus", "questa", "modelsim", "ius", "xcelium", "vcs", "ghdl", "riviera", "activehdl", "cvc"]
+    if simulator not in supported_sim:
+        raise ValueError("Wrong simulator name. Supported: " + ", ".join(supported_sim))
+
+    if simulator_name == "questa":
+        simulator_name = "modelsim"
+
+    if simulator_name == "xcelium":
+        simulator_name = "ius"
+
+    if simulator_name == "cvc":
+        simulator_name = "modelsim"
+
+    if simulator_name in ["riviera", "activehdl"]:
+        simulator_name = "aldec"
+
+    lib_prefix = "lib"
+
+    lib_ext = ".so"
+    if os.name == "nt":
+        lib_ext = ".dll"
+
+    if simulator_name == "icarus":
+        lib_ext = ""
+
+    # check if compiled with msvc
+    if os.path.isfile(os.path.join(libs_dir, "cocotb.dll")):
+        lib_prefix = ""
+
+    return lib_prefix + "cocotb" + interface_name + "_" + simulator_name + lib_ext
```

Comment:
Slightly clearer as:
```suggestion
    if simulator_name == "icarus":
        lib_ext = ""
    elif os.name == "nt":
        lib_ext = ".dll"
    else:
        lib_ext = ".so"

    # check if compiled with msvc
    if os.path.isfile(os.path.join(libs_dir, "cocotb.dll")):
        lib_prefix = ""
    else:
        lib_prefix = "lib"

    return lib_prefix + "cocotb" + interface_name + "_" + simulator_name + lib_ext
```